Lib/threading.py

```diff
@@ -1475,11 +1482,34 @@ def __init__(self):
         with _active_limbo_lock:
             _active[self._ident] = self
 
+# Helper thread-local instance to detect when a _DummyThread
+# is collected. Not a part of the public API.
+_thread_local_info = local()
+
```

Comment:
I wonder why the linter action hasn't gotten that... Anyways, just fixed.

Reply:
I guess a lot of existing code doesn't adhere to PEP 8, so we can not check it in the GHA. But since these codes have been added in this PR, we should adhere to it.